clients/web/static/lib/bootstrap/css

```diff
@@ -1 +0,0 @@
-../../built/
```

Comment:
Nevermind, just realized why. So, in the end, this will in fact break any app that hasn't included `ext.min.css`, but that sort of needs to be included anyway at this point.